db/migrate/20191007100156_add_reminder_consent_to_patients.rb

```diff
@@ -0,0 +1,5 @@
+class AddReminderConsentToPatients < ActiveRecord::Migration[5.1]
+  def change
+    add_column :patients, :reminder_consent, :string, index: true, null: false, default: Patient.reminder_consents[:denied]
```

Comment:
Maybe make this `concurrently` since this will re-write the whole table with the default value.